python/pyproject.toml

```diff
@@ -50,9 +50,7 @@ build-backend = "poetry.core.masonry.api"
 line-length = 100
 target-version = ["py39"]
 include = ["graphframes"]
-
-[tool.flake8]
```

Comment:
apache spark does use I think its 88.. but why do you remove it?

Reply:
flake8 does not support pyproject.toml; I added tox.ini (copied from [spark](https://github.com/apache/spark/blob/master/dev/tox.ini))